src/content/helpers/maps.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable camelcase */
+export default {
+  de_dust2: 'Dust2',
+  de_mirage: 'Mirage',
+  de_overpass: 'Overpass',
+  de_inferno: 'Inferno',
+  de_nuke: 'Nuke',
+  de_cache: 'Cache',
+  de_train: 'Train',
+  de_vertigo: 'Vertigo'
+}
```

Comment:
Import naming could be made more consistent:
```suggestion
export default {
	csgo: {
	  de_dust2: 'Dust2',
	  de_mirage: 'Mirage',
	  de_overpass: 'Overpass',
	  de_inferno: 'Inferno',
	  de_nuke: 'Nuke',
	  de_cache: 'Cache',
	  de_train: 'Train',
	  de_vertigo: 'Vertigo'
  }
}
```

```ts
import maps from '../helpers/maps';

// maps.csgo
```